### ISSUE TITLE
508-defect-1 ❗️ Add an issue page is not responsive

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#53445]
### Point of contact

@joshkimux

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

Add an issue page is not responsive when zoomed on mobile.

<img width="748" alt="Screenshot 2023-02-09 at 3 40 58 PM" src="https://user-images.githubusercontent.com/14154792/217933906-9c8e8024-dd28-44e3-b0fc-149cde2375dc.png">

### Reproduction steps

1. [Login using the instructions here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/52046)
2. Reduce the screenwidth to 360px and zoom 200x. Confirm the page clips.

### Proposed solution or next steps

I'm unsure what the source of the issue is, so it's difficult to provide a proposed solution or immediate next steps aside from assigning to an engineer to investigate.

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [X] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by Mottie** [comment]
This is actually a [problem with the `va-date` web component](https://dsva.slack.com/archives/CBU0KDSB1/p1674589851200819). After mentioning that the date overflows on 320 px screens, it was suggested that we switch to use the `va-memorable-date` web component. So that is what I'll do to fix this.

**Comment by Mottie** [comment]
Changes have been merged and are now available in staging for testing.

**Comment by joshkimux** [comment]
<img width="719" alt="Screenshot 2023-02-21 at 1 25 45 PM" src="https://user-images.githubusercontent.com/14154792/220428597-93c24f2c-c5e3-4534-b0da-5231f25b4ab8.png">

Confirming this has been fixed 🥳 Y'all are on fire 🔥
